<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>timer</string>
  </property>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>35</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Resource/bg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Days1">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>80</y>
     <width>28</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Days2">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>80</y>
     <width>28</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Days3">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>80</y>
     <width>28</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Hours1">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>80</y>
     <width>28</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Hours2">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>80</y>
     <width>28</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Minutes1">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>80</y>
     <width>28</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Minutes2">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>80</y>
     <width>28</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Seconds1">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>80</y>
     <width>28</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Seconds2">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>80</y>
     <width>28</width>
     <height>49</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>距离2022考研还有</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>80</y>
     <width>12</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>80</y>
     <width>12</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>80</y>
     <width>12</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
